docs/src/main/asciidoc/migration-guide-1.x.adoc

```diff
@@ -0,0 +1,37 @@
+== Migration Guide from Spring Cloud GCP 1.x
+
+=== Maven Group ID Change
+Spring Cloud  link:https://spring.io/blog/2019/07/24/simplifying-the-spring-cloud-release-train[unbundled] Spring Cloud GCP and other IaaS providers from their release train.
+To use the newly unbundled libraries, change the group IDs in your `pom.xml` files:
+
+[source,xml]
+.pom.xml
+----
+<dependency>
+  <groupId>com.google.cloud</groupId> <!-- formerly org.springframework.cloud</groupId> -->
+  <artifactId>spring-cloud-gcp-starter-config</artifactId>
```

Comment:
Maybe we should display the dependency of our BOM here instead? Or maybe just a different one; apparently the GCP Runtime Config API does not have plans of going out of beta.

Reply:
Changed to `spring-cloud-gcp-dependencies`